docs/deviceParameter.md

```diff
@@ -0,0 +1,141 @@
+
+## DeviceParameter 
+
+The `DeviceParameter` type is a subType of [`deviceEvent`](https://developer.tidepool.org/data-model/device-data/types/deviceEvent/). 
+
+It gives details on the parameter, its current value and its previous value. 
+
+```json
+{
+  "type": "DeviceEvent",
+  "subType": "DeviceParameter",
+  "time": "2020-01-20T08:17:07.920Z",
+  "timezoneOffset": 60,
+  "name": "Name of the Parameter",
+  "value": "Current value of the parameter",
+  "units": "%",
+  "lastUpdateDate": "2020-01-20T08:10:00.000Z",
+  "previousValue": "Previous value of the parameter",  
+  "level": "1",
+  "minValue": "0",
+  "maxValue": "10",
+  "processed": "True",
+  "linkedSubType": ["DeviceParameterName1", "DeviceParameterName2"]
+} 
+```
+
+### name
+
+Name of the device parameter.
+
+```
+QUICK SUMMARY
+Required:
+    platform: yes
+```
+
+### value
+
+Value of the device parameter.
+
+```
+QUICK SUMMARY
+Required:
+    platform: yes
+```
+
+### units
+
+Units that is used for this parameter. 
+
+```
+QUICK SUMMARY
+Required:
+    platform: yes
```

Comment:
Some parameters don't have any unit but the field is required ? 
Maybe we should have a default value when unit is N/A.